source/js/components/people/person.jsx

```diff
@@ -3,16 +3,112 @@ import React from 'react';
 export default class Person extends React.Component {
   constructor(props) {
     super(props);
+    this.flip = this.flip.bind(this);
+
+    this.state = {
+      flipped: false
+    };
+
+  }
+
+  flip() {
+    this.setState({'flipped': !this.state.flipped});
   }
 
   render() {
-    return (
-      <div className="col-lg-3 p-3 person">
-        <h4>{this.props.metadata.name}</h4>
-        <p>👤 {this.props.metadata.role}</p>
-        <p>▼ {this.props.metadata.location}</p>
-        <p>&hearts;&nbsp;<small>{this.props.metadata.teams[0]}</small></p>
-      </div>
-    );
+
+
+    let issues = this.props.metadata.internet_health_issues.map((issue, index) => {
+      return (
+        <a href="#" key={index} className="link-small issue-link d-inline-block mr-1">{issue}</a>
+      );
+    });
+
+    let socialLinks = Object.keys(this.props.metadata.links).map((linkKey, index) => {
+      let classes = linkKey + ` mr-3`;
+
+      return (
+        <a href={this.props.metadata.links[linkKey]} className={classes} key={index}></a>
+      );
+    });
+
+    if (this.props.metadata.featured) {
+      return (
+        <div className="col-12 p-3 mb-4">
+          <div className="person-card person-card-featured row no-gutters">
+            <div className="col-12 col-sm-4 mr-3">
+              <div className="row">
+                <div className="col">
+                  <img src={this.props.metadata.image} alt="Headshot" />
```

Comment:
Makes sense, but I changed the column classes to (always 4 columns, even at xs) because we were getting *giant* faces at xs sizes.